File: cakk-api/src/main/java/com/cakk/api/dto/request/link/UpdateLinkRequest.java
package com.cakk.api.dto.request.link;

import jakarta.validation.constraints.Size;

public record UpdateLinkRequest(
	@Size(min = 1, max = 200)
	String instagram,
	@Size(min = 1, max = 200)
	String kakao,
	@Size(min = 1, max = 200)
	String web
) {
}

File: cakk-api/src/main/java/com/cakk/api/dto/request/link/UpdateLinkRequest.kt
package com.cakk.api.dto.request.link

import jakarta.validation.constraints.Size

data class UpdateLinkRequest(
	@field:Size(min = 1, max = 200)
	val instagram: String?,
	@field:Size(min = 1, max = 200)
	val kakao: String?,
	@field:Size(min = 1, max = 200)
	val web: String?
)
